packages/mermaid/src/diagrams/flowchart/flowDb.js

```diff
@@ -192,6 +192,12 @@ export const updateLinkInterpolate = function (positions, interp) {
  */
 export const updateLink = function (positions, style) {
   positions.forEach(function (pos) {
+    if (pos >= edges.length) {
```

Comment:
We should have a testcase to make sure a non-TypeError is thrown.

Reply:
Thanks for this suggestion, Added the new test.